packaging/vagrant/Vagrantfile

```diff
@@ -262,7 +305,6 @@ EOF2
   # Note: This will grow the image quite a bit. Don't run it before taking the
   # snapshot to be distributed to end users.
   c.vm.provision "test", type: "shell", run: "never", privileged: false,
-                 env: { "CH_TEST_SCOPE" => "full" },
```

Comment:
I believe the original purpose here was to inherit `CH_TEST_SCOPE` from `vagrant`'s environment. I'm not sure that's still a good idea, but FWIW.